docs/discover/save-search.asciidoc

```diff
@@ -3,19 +3,18 @@
 
 A saved search is a convenient way to reuse a search
 that you've created in *Discover*.
-Saved searches are good for adding search results to a dashboard.
+Saved searches are good for adding search results to a dashboard,
+and can also serve as a foundation for building visualizations.
 
 [role="screenshot"]
 image::discover/images/saved-search.png[Example of Discover's save search option]
 
 
-A saved search stores the query text, filters, and
+A saved search stores the query text, filters, time range, and
 current view of *Discover*&mdash;the columns selected in the document table,
 the sort order, and the index pattern.
-
-Saved searches are different from <<save-load-delete-query,saved queries>>, which store
-only the query and are available in any app that has a query bar. Saved searches
-can serve as a foundation for building visualizations.
+Saved searches are different from a <<save-load-delete-query,saved queries>>, which store
+just the query and are available in any app with a query bar.
```

Comment:
Well, just the query ... and filters ... and time filter  (if wanted)
<img width="544" alt="Bildschirmfoto 2021-04-26 um 15 45 02" src="https://user-images.githubusercontent.com/463851/116092851-730dd380-a6a6-11eb-9fc4-e2907df10f35.png">